Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.py

```diff
@@ -0,0 +1,299 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+from typing import Dict, Any, List, Optional, Tuple
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+VENDOR = 'Threat Crowd'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+DEFAULT_RESOLUTION_LIMIT = 10
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, base_url: str, verify: bool, proxy: bool, reliability: DBotScoreReliability, extended_data: bool):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self.reliability = reliability
+        self.extended_data = extended_data
+
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_resolutions(resolutions: List[dict], limit: Optional[int]) -> List[dict]:
+    """ Gets a resolution section from response.
+     return a limited sorted list, desc by time.
```

Comment:
what does "limited" mean here?
Also, I suggest we stick to google docstring format, here and everywhere